feedback.md

```diff
@@ -0,0 +1,32 @@
+Hi Kim!
+
+Many thanks for this awesome submission. I went through all (I hope) files and have a few follow-up questions - no coding needed, just conceptual stuff!
+
+Hope that's fine! :)
+
+# Worker:
+
+https://github.com/Burgestrand/kbc19/blob/master/lib/tasks/active_job.rake all good here! Clean and concise!
+
+# Jobs:
+
+https://github.com/Burgestrand/kbc19/blob/master/app/jobs/application_job.rb#L5: we may want to file this as a bug in rails/rails; if I remember correctly connection pool errors during deserialize would lead to discard!
+
+# Adapter:
+
+https://github.com/Burgestrand/kbc19/blob/master/app/jobs/google_pub_sub_adapter.rb
+
+- Maybe should go to lib? Unsure.
+- Naming: Queue makes sense to way we use it!
+- I would suggest to put execution into the worker, so the adapter only deals with enqueueing
```

Comment:
Yeah, I agree with this.

Originally had most of the code in the Rake task, but moved execution into the adapter for easier testing. Next step is probably moving it to a separate class, and thus a separate file.

There's some entanglement in that the client (i.e. enqueuer) and worker (i.e. dequeuer) need to agree on the serialization format. I'd probably recommend putting the serialize/deserialize into one file (to keep those two together), and then make the client and worker use that. That'd be their bridge!